src/chroot.c

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
move this code to `unistd.c`, so that we don't have one file per system call,but grouped by subject/header (roughly)